api/v1alpha1/packagebundlecontroller_types.go

```diff
@@ -85,6 +85,11 @@ type PackageBundleControllerSpec struct {
 	// Repository portion of an OCI address to the bundle
 	// +optional
 	BundleRepository string `json:"bundleRepository"`
+
```

Comment:
From a design perspective will we also have the responsibility to remove the namespace if we create it?

Reply:
No we should not delete the namespace that was created. It will be left there as it's currently implemented.